src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -53,4 +52,8 @@ public void addSection(Section section) {
     public void remove(Station station) {
         sections.removeSection(this, station);
     }
+
+    public List<Section> getAllSection() {
+        return sections.getSections();
```

Comment:
요건 외부에서 참조를 받아 add하거나 하는 경우 `Sections`의 데이터 정합성이 깨질 수 있겠네요

아래와 같이 반환하는건 어떨까요?

```suggestion
        return Collections.unmodifiablelistsections.getSections());
```

Reply:
객체를 좀 더 안전하게 Immutable로 선언해서 사용하면 의도하지 않은 변경이 일어나지 않겠네요.
좋은 피드백 감사드립니다!